strapi/strapi
# Refactoring the filter in CM

### What does it do?

It modifies the filter in the CM to show something that looks like the media library filters

### Why is it needed?

Consistency

### How to test it?

Go to the CM and use the filter. You should see a dropdown down with two selects and an input. When validating, the URL should change with the good filter
